app/src/main/java/io/homeassistant/companion/android/sensors/PhoneStateSensorManager.kt

```diff
@@ -105,18 +110,12 @@ class PhoneStateSensorManager : SensorManager {
                 }
             }
 
-            if (slotIndex == 1) {
-                return sim_2.toSensorRegistration(
-                    displayName,
-                    "mdi:sim",
-                    attrs
-                )
-            }
-            return sim_1.toSensorRegistration(
+            onSensorUpdated(context,
+                sim_1,
```

Comment:
There should be `basicSimSensor` i'm not mistaken

Reply:
Good catch!